helpers/ban_manager.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+/**
+ * Temporarily peers banning mechanism.
+ *
+ * @class
+ * @memberof helpers
+ * @param {Logger} logger
+ * @param {Object} config - app configuration
+ * @param {function} unbanPeer - app configuration
+ */
+function BanManager(logger, config, unbanPeer) {
```

Comment:
Why we pass a function here? That seems a little weird to me, can we go with different approach?